modules/web/client/src/clue/scala/observe/queries/VisitQueriesGQL.scala

```diff
@@ -31,10 +32,10 @@ object VisitQueriesGQL:
     // ${ExecutionVisitsSubquery.Fragments}
 
   @GraphQL
-  trait UpdateDatasetQAState extends GraphQLOperation[ObservationDB]:
+  trait UpdateDatasetQa extends GraphQLOperation[ObservationDB]:
```

Comment:
QA?

Reply:
Not sure how else to group `qaState` + `comment`, so I called it `Qa`.